e2e/specs/login.e2e-spec.ts

```diff
@@ -0,0 +1,25 @@
+import { browser } from 'protractor';
+import { LoginPage } from '../pages/login.po';
+import { Credentials } from '../configs/credentials';
+let loginPage: LoginPage
+
+describe('Login', () => {
+  beforeEach(() => {
+    loginPage = new LoginPage();
+  });
+  describe('When login using invalid credentials', () => {
+    it('should not change URL', () => {
+      const expectedURL = browser.getCurrentUrl()
+      loginPage.login('invalidUserName', 'invalidPassword')
+      browser.waitForAngular()
```

Comment:
Do we need to explicitly call this function? Are we waiting for the page to redirect? If so, could we use Protractor's 'Expected Conditions' ? <Note: I haven't used EC from Protractor, I just came across a post on stackoverflow which recommends it for such cases> https://stackoverflow.com/questions/29397800/how-can-i-get-the-current-url-using-protractor